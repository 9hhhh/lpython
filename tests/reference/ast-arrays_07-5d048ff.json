{
    "basename": "ast-arrays_07-5d048ff",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/arrays_07.f90",
    "infile_hash": "43140ddb20889ad30943c7abe8f223591b5f7e6a04cfcbfaf1ad5e3e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-arrays_07-5d048ff.stdout",
    "stdout_hash": "57667eb5abbfd6d472ef54ba1e200fb17c5894b08a8c0dae895dcb8d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}