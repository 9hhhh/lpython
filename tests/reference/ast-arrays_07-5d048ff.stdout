<<<<<<< HEAD
(TranslationUnit [(Program arrays_07 [] [(ImplicitNone [])] [(Declaration (AttrType TypeInteger [] ()) [(AttrDimension [(1 10 DimensionExpr)])] [(x [] ())]) (Declaration (AttrType TypeInteger [] ()) [(AttrDimension [(1 10 DimensionExpr) (1 10 DimensionExpr)])] [(y [] ())]) (Declaration (AttrType TypeInteger [] ()) [] [(i [] ()) (j [] ()) (start [] ()) (end [] ()) (step [] ())])] [(DoLoop 0 () i 1 10 () [(DoLoop 0 () j 1 10 () [(= 0 (FuncCallOrArray y [] [(() i ()) (() j ())] []) (+ i j))])]) (= 0 x (ArrayInitializer [(ImpliedDoLoop [i] i 1 10 ())])) (= 0 start (FuncCallOrArray x [] [(() 4 ())] [])) (= 0 end (FuncCallOrArray x [] [(() 7 ())] [])) (= 0 step (FuncCallOrArray x [] [(() 1 ())] [])) (Print 0 () [(FuncCallOrArray x [] [(start end step)] [])]) (Print 0 () [x]) (Print 0 () [(FuncCallOrArray x [] [(() () 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 5 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 () 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() 5 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 5 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() 5 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 () 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 () 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() () 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() () 2)] [])]) (Print 0 () [y]) (Print 0 () [(FuncCallOrArray y [] [(() () 1) (() 3 ())] [])]) (Print 0 () [(FuncCallOrArray y [] [(3 5 1) (() () 1)] [])]) (Print 0 () [(FuncCallOrArray y [] [(4 () 1) (3 () 1)] [])]) (Print 0 () [(FuncCallOrArray y [] [(() 5 1) (5 () 1)] [])]) (Print 0 () [(FuncCallOrArray y [] [(3 5 2) (3 5 1)] [])]) (Print 0 () [(FuncCallOrArray y [] [(() 5 2) (() 6 ())] [])]) (Print 0 () [(FuncCallOrArray y [] [(() 4 ()) (3 () 2)] [])]) (Print 0 () [(FuncCallOrArray y [] [(3 () 2) (() 3 2)] [])]) (Print 0 () [(FuncCallOrArray y [] [(() () 2) (() () 4)] [])]) (Print 0 () [(FuncCallOrArray y [] [(() () 2) (() 5 ())] [])])] [])])
=======
(TranslationUnit [(Program arrays_07 [] [(ImplicitNone [])] [(Declaration (AttrType TypeInteger [] ()) [(AttrDimension [(1 10 DimensionExpr)])] [(x [] [] ())]) (Declaration (AttrType TypeInteger [] ()) [] [(i [] [] ())])] [(= 0 x (ArrayInitializer () [(ImpliedDoLoop [i] i 1 (FuncCallOrArray size [] [(() x ())] []) ())])) (Print 0 () [x]) (Print 0 () [(FuncCallOrArray x [] [(() () 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 5 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 () 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() 5 1)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 5 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() 5 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 () 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(3 () 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() () 2)] [])]) (Print 0 () [(FuncCallOrArray x [] [(() () 2)] [])])] [])])
>>>>>>> master
